main.py

```diff
@@ -1,7 +1,34 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    [Genre.objects.create(name=name) for name in ["Western",
```

Comment:
Huh. This style doesn't feel quite right to me. I would certainly use a traditional loop syntax here.